prov/rxm/src/rxm_ep.c

```diff
@@ -1044,12 +1044,12 @@ rxm_ep_msg_normal_send(struct rxm_conn *rxm_conn, struct rxm_pkt *tx_pkt,
 	return fi_send(rxm_conn->msg_ep, tx_pkt, pkt_size, desc, 0, context);
 }
 
-static ssize_t
-rxm_ep_alloc_rndv_tx_res(struct rxm_ep *rxm_ep, struct rxm_conn *rxm_conn,
-			 void *context, uint8_t count, const struct iovec *iov,
-			 void **desc, size_t data_len, uint64_t data,
-			 uint64_t flags, uint64_t tag, uint8_t op,
-			 struct rxm_tx_rndv_buf **tx_rndv_buf)
+static inline ssize_t
```

Comment:
Compilers will ignore inline in .c files.  Additionally, this is a long function to attempt to inline, unless it's only called from one place, in which case the compiler will inline anyway.

Reply:
Will fix.